model-optimizer/extensions/middle/SplitConcatPairToInterpolate.py

```diff
@@ -96,12 +99,21 @@ def replace_interpolate_pattern(graph: Graph, match: dict):
                                                      })
     shape_node.out_port(0).connect(strided_slice_node.in_port(0))
 
-    strided_slice_node.out_port(0).connect(mul_node.in_port(0))
+    cast_shape_to_float = Cast(graph, {'dst_type': np.float32}).create_node()
```

Comment:
Why do we need this? After this change the "sizes" input of Interpolate will have floating point type. But it should be integer, right? So the conversion to float is really needed than it is necessary to convert number to int64 before passing to Interpolate

Reply:
Fixed.